src/SBInterpolatedImage.cpp

```diff
@@ -864,20 +1516,18 @@ namespace galsim {
         return Position<double>(_xcentroid, _ycentroid);
     }
 
+    // Not used anymore.
     ConstImageView<double> SBInterpolatedKImage::SBInterpolatedKImageImpl::getKData() const
     {
-        int N = _ktab->getN();
-        dbg << "_ktab->getN(): " << N << std::endl;
-        double *data = reinterpret_cast<double*>(_ktab->getArray());
-        // JM - I'm not completely confident that I got the dimensions below correct,
-        //      (i.e., should it be (2N x N/2+1) or (N/2+1 x 2N)?), but it doesn't
-        //      actually matter for the intended application, which is just to store and
-        //      later retrieve the 2N * N/2+1 memory-contiguous numbers representing the
-        //      KTable.
+        double* data = const_cast<double*>(reinterpret_cast<const double*>(_kimage.getData()));
+        int No2 = _kimage.getBounds().getXMax();
+        int N = 2*No2;
         return ConstImageView<double>(data, shared_ptr<double>(), 1, 2*N,
-                                      Bounds<int>(0,2*N-1,0,N/2));
+                                      Bounds<int>(0,N+1,-No2,No2-1));
     }
 
+    // Not used anymore.
+    // MJ: Maybe time to remove all these SBProfile serialize functions?
```

Comment:
Fine with me if they're not being used :)

Reply:
Good.  Done.